Fika.Core/Coop/Utils/FikaBackendUtils.cs

```diff
@@ -32,7 +35,15 @@ public static class FikaBackendUtils
         public static int LocalPort = 0;
         public static bool IsHostNatPunch = false;
         private static string groupId;
+        private static long timestamp;
         private static string raidCode;
+        public static bool IsReconnect = false;
+        public static bool ReconnectPacketRecieved = false;
+        public static ReconnectResponsePacket? ReconnectPacket; // change to not be nullable
+        public static bool SpawnedPlayersComplete = false;
+        public static List<AirdropLootPacket> OldLootPackets = [];
+        public static List<AirdropPacket> OldAirdropPackets = [];
+        public static List<FikaAirdropBox> OldAirdropBoxes = [];
```

Comment:
I think all these new fields should be cleaned up at a later stage, possibly just adding a new class and containing them in there. We can then `null` or `Dispose()` that class later and clean up the entire reconnect data. In general I try to avoid as many `static` as possible.

Reply:
completely agree, can make that a 2.0 if you'd like